charts/pulsar/values.yaml

```
@@ -731,6 +732,7 @@ broker:
 ##
 functions:
   component: functions-worker
+  useBookieAsStateStore: true
```

```suggestion
  useBookieAsStateStore: false
```
Let's disable this by default.